.github/workflows/lint-test-sdk.yml

```diff
@@ -50,17 +51,6 @@ jobs:
   run-tests-with-tox:
     runs-on: ubuntu-latest
     steps:
-      - uses: "actions/checkout@v2"
```

Comment:
extracted into a composite action. Keeps the main workflow file cleaner in exchange for some more boilerplate and yet another yaml file